SofaKernel/modules/SofaHelper/src/sofa/helper/ComponentChange.cpp

```diff
@@ -458,7 +458,7 @@ std::map<std::string, ComponentChange> uncreatableComponents = {
 
     // SofaEngine was pluginized in #1603
     { "BoxROI", Pluginized("v20.12", "SofaEngine") },
-
+    
```

Comment:
info regarding EulerSolver pluginized  is missing

Reply:
👍